scss/abstracts/variables/_font.scss

```diff
@@ -0,0 +1,17 @@
+$base: 18px; // used to set rem to 18px
+
+// ---------------------------------------------------- Font sizes -----------------------------------------------------
+$font-size-s: 0.7rem;
+$font-size-m: 1rem;
```

Comment:
Where did you get these sizes from?

The type scale we are using is responsive: https://ons-design-system.netlify.app/styles/typography/

Reply:
Will grab the ones from there.